Packs/FeedORKL/pack_metadata.json

```diff
@@ -0,0 +1,23 @@
+{
+    "name": "ORKL Threat Intel Feed",
+    "description": "Ingest indicators from the ORKL feed.",
+    "support": "xsoar",
```

Comment:
@mohlcyber Just to verify, which support level would you like? xsoar, or community.

Reply:
Hi @amshamah419 good point. Will change it to community. Long term we can check if XSOAR support is needed.

Reply:
Seems i am not able to update the branch. @amshamah419 could you please check if you can change the support from xsoar to community please?

Reply:
@mohlcyber I can try to pull the branch on your fork and update it there.